src/C-DEngine/Interfaces/cde_ICDEKpis.cs

```diff
@@ -57,7 +57,7 @@ public enum eKPINames //NOSONAR - CDE API Rule
         /// <summary>
         /// The blobs not found
         /// </summary>
-        BlobsNotFound, RejectedClientConnections, IISRejectedClientConnections,
+        BlobsNotFound, RejectedClientConnections, IISRejectedClientConnections, WsClientConnectErrors
```

Comment:
How about creating a new comment segment "/// connnection issues" and move all but BlobsNotFound in that segmet?

Reply:
Good idea, changed that...